### PR TITLE
refactor: use RFC 176 style module imports

### PR DESCRIPTION
**Pull request by buschtoens**
https://github.com/ember-cli/ember-rfc176-data

### CODE REVIEW

File: addon/services/raven.js [hunk 1]
```diff
@@ -1,17 +1,12 @@
+import { assign } from '@ember/polyfills';
```
Comment (dschmidt) [inline addon/services/raven.js]: Does this handle backwards-compatibility like the old code?

File: addon/services/raven.js [hunk 2]
```diff
@@ -126,7 +123,7 @@ export default Service.extend({
    * @throws {Error} An error if Raven cannot capture the exception
    */
   captureException(error) {
-    if (this.get('isRavenUsable')) {
+    if (get(this, 'isRavenUsable')) {
```
Comment (Turbo87) [inline addon/services/raven.js]: why did you change all those getters??

File: addon/services/raven.js [hunk 3]
```diff
@@ -100,12 +95,14 @@ export default Service.extend({
...
-        release: this.get(serviceReleaseProperty) || config.APP.version
+        release: get(this, serviceReleaseProperty) || config.APP.version
       }, ravenOptions);
 
       Raven.config(dsn, ravenConfig);
     } catch (e) {
-      Ember.Logger.warn('Error during `sentry` initialization: ' + e);
+      warn('Error during `sentry` initialization: ' + e, {
+        id: 'ember-cli-sentry.initialization-error'
+      });
```
Comment (Turbo87) [inline addon/services/raven.js]: adding an `id` here seems unrelated to the PR goal of moving towards RFC176 imports

File: addon/services/raven.js (hunk 1 above)
Comment (Turbo87) [reply addon/services/raven.js]: this PR should only refactor to use RFC176 imports, but not change any behavior. can you adjust the code to preserve the old behavior?

File: tests/test-helper.js [hunk 4]
```diff
@@ -1,7 +1,5 @@
 import resolver from './helpers/resolver';
-import {
-  setResolver
-} from 'ember-qunit';
+import { setResolver } from '@ember/test-helpers';
```
Comment (Turbo87) [inline tests/test-helper.js]: this seems unrelated to the RFC176 imports
